### PR TITLE
Enable windows build

### PR DESCRIPTION
**Pull request by dschwoerer**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by dschwoerer** [comment]
@johnomotani I am trying to enable windows builds - do you know how to do that?
I think I have a patch to get it building on windows ...

**Comment by johnomotani** [comment]
I don't know - I'd have expected the change you made to do it!

**Comment by dschwoerer** [comment]
I had to run conda-smithy.
Rather then spending so much time to get it running with python3.12 I could have just use a magic comment as mentioned in the template :see_no_evil:

**Comment by dschwoerer** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,11 +8,12 @@ package:
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/eudist-{{ version }}.tar.gz
   sha256: 1f0805eb0af592907584b10ebefb6c1636d34d7ff0e08726515176e55200000c
+  patches:
+    - "windows.patch" # the patch file is expected to be found in the recipe
 
 build:
   script: {{ PYTHON }} -m pip install . -vv
   number: 0
```
Comment (dschwoerer) [inline recipe/meta.yaml:16]:
```suggestion
  number: 1
```

**Comment by dschwoerer** [comment]
@conda-forge-admin, please rerender

**Pull request merged**